R/metadata.R

```diff
@@ -46,10 +54,17 @@ setClass(
     load.rate.units="character",
     # Other
     station="character",
+    wq.sta.id="character",
+    station.lat="ANY",
```

Comment:
Yeah, I originally set them as numeric, it was causing problems though ( I think with the default argument they get set as in the prototype list).